### PR TITLE
Added Sarah attempt 2

### PR DESCRIPTION
**Pull request by trysarahtops**
Updated sf people file

### CODE REVIEW

**Comment by gonuke** [comment]
It looks like all the changes in #244 were correct, but were undone by your revert PR #245?  This PR is missing the changes to `_data/people.yml` again.

**Comment by lewisgross1296** [comment]
This is looking good again, but perhaps we should clean up the history with the revert? I don't think it hurts anyone to leave it though. @gonuke let us know your thoughts

**Review by gonuke — CHANGES_REQUESTED** [review]
I've tested this and remembered that the image is resized to a square aspect ratio on the personal page.  That means it's best to have an image that is close to square.  Thus, I recommend cropping the photo to be closer to square.

File: community/people/sf/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: person
+image: Sarah.png
```
Comment (gonuke) [inline community/people/sf/index.md:3]: I think you'll need to update this for the new image filename, too

**Comment by trysarahtops** [comment]
Made the image more square, tried to test and got an error.
```
sgfrancis@cnerg-docker-05:~/cnerg.github.com (add_myself) $ docker run --rm --volume="$PWD:/srv/jekyll" --volume="$PWD/vendor/bundle:/usr/local/bundle" -p 4000:4000  jekyll/jekyll  jekyll serve
docker: permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Head "http://%2Fvar%2Frun%2Fdocker.sock/_ping": dial unix /var/run/docker.sock: connect: permission denied.
See 'docker run --help'.
```

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for updating the image - I think you need to update one more reference to that image

**Comment by gonuke** [comment]
> Made the image more square, tried to test and got an error.
> 
> ```
> 
> sgfrancis@cnerg-docker-05:~/cnerg.github.com (add_myself) $ docker run --rm --volume="$PWD:/srv/jekyll" --volume="$PWD/vendor/bundle:/usr/local/bundle" -p 4000:4000  jekyll/jekyll  jekyll serve
> 
> docker: permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Head "http://%2Fvar%2Frun%2Fdocker.sock/_ping": dial unix /var/run/docker.sock: connect: permission denied.
> 
> See 'docker run --help'.
> 
> ```

Have you run docker on this machine before?  I think there is a command that is needed before the first time, and I can't recall if you've done that before?

**Comment by trysarahtops** [comment]
I entered  `cae-docker-setup`. It runs now but I got an error.
```
sgfrancis@cnerg-docker-05:~/cnerg.github.com (add_myself) $ docker run --rm --volume="$PWD:/srv/jekyll" --volume="$PWD/vendor/bundle:/usr/local/bundle" -p 4000:4000  jekyll/jekyll  jekyll serve
Unable to find image 'jekyll/jekyll:latest' locally
latest: Pulling from jekyll/jekyll
df9b9388f04a: Pull complete 
837e9cfc7e43: Pull complete 
c7850f1a8c23: Pull complete 
6ca4c39baa3d: Pull complete 
daa3a8cb79d3: Pull complete 
cf5821b9dd48: Pull complete 
Digest: sha256:400b8d1569f118bca8a3a09a25f32803b00a55d1ea241feaf5f904d66ca9c625
Status: Downloaded newer image for jekyll/jekyll:latest
chown: /usr/local/bundle: Operation not permitted
chown: /usr/local/bundle: Operation not permitted
There was an error while trying to write to `/usr/local/bundle/config`. It is
likely that you need to grant write permissions for that path. 
```

**Comment by gonuke** [comment]
> I entered `cae-docker-setup`. It runs now but I got an error.
> 
> ```
> sgfrancis@cnerg-docker-05:~/cnerg.github.com (add_myself) $ docker run --rm --volume="$PWD:/srv/jekyll" --volume="$PWD/vendor/bundle:/usr/local/bundle" -p 4000:4000  jekyll/jekyll  jekyll serve
> Unable to find image 'jekyll/jekyll:latest' locally
> latest: Pulling from jekyll/jekyll
> df9b9388f04a: Pull complete 
> 837e9cfc7e43: Pull complete 
> c7850f1a8c23: Pull complete 
> 6ca4c39baa3d: Pull complete 
> daa3a8cb79d3: Pull complete 
> cf5821b9dd48: Pull complete 
> Digest: sha256:400b8d1569f118bca8a3a09a25f32803b00a55d1ea241feaf5f904d66ca9c625
> Status: Downloaded newer image for jekyll/jekyll:latest
> chown: /usr/local/bundle: Operation not permitted
> chown: /usr/local/bundle: Operation not permitted
> There was an error while trying to write to `/usr/local/bundle/config`. It is
> likely that you need to grant write permissions for that path. 
> ```

I'm not sure why you're getting that error ??  but maybe try without the second "--volume" command-line argument?

**Comment by gonuke** [comment]
FWIW, the image is still a little distorted

**Comment by Edgar-21** [comment]
built locally, lgtm

**Pull request merged**